aiven/user_config.go

```diff
@@ -381,6 +398,20 @@ func convertTerraformUserConfigValueToAPICompatibleFormat(
 			omit = true
 		}
 	case "boolean":
+		switch value.(type) {
+		case string:
+			if value == "<<value not set>>" {
+				omit = true
+			} else {
+				b, err := strconv.ParseBool(value.(string))
+				if err != nil {
+					panic(fmt.Sprintf("Invalid %v user config key type %T for %v, expected string", serviceType, value, key))
```

Comment:
Error message looks incorrect. The type is string but value is not something that can be parsed as bool.